detection_rules/etc/version.lock.json

```diff
@@ -8107,15 +8121,15 @@
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Clearing Windows Console History",
-        "sha256": "5a2ba281f21edbffcdc9934e0317b8d2ca9b0499c0c4954f0987a3a41bab5fd5",
+        "sha256": "38d90e293ad91df8f5b8d1b50f36ccf4ae6d4c025e1a72f7b44ee1c8cb296950",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.